selfdrive/ui/layouts/settings/developer.py

```diff
@@ -16,34 +16,33 @@
   ),
 }
 
-
 class DeveloperLayout(Widget):
```

Comment:
```suggestion

class DeveloperLayout(Widget):
```

two lines around functions/classes